app/views/articles/_sidebar.html.erb

```diff
@@ -7,6 +7,9 @@
     <% end %>
     <div class="js-display-ad-container" data-async-url="<%= billboard_path(placement_area: :sidebar_left) %>"></div>
     <div class="js-display-ad-container pt-4" data-async-url="<%= billboard_path(placement_area: :sidebar_left_2) %>"></div>
+    <% cache("footer-modal-#{user_signed_in?}-#{ForemInstance.deployed_at}-#{Settings::General.admin_action_taken_at&.rfc3339}", expires_in: 24.hours) do %>
```

Comment:
Does the footer need to know if the user is signed in? I think you can omit the `user_signed_in` portion, else we would have two set of cache of the same content.

Reply:
Done. Removed the `sign-in` info as it is not required.